docs_user/modules/proc_stopping-infrastructure-management-and-compute-services.adoc

```diff
@@ -2,23 +2,21 @@
 
 = Stopping infrastructure management and Compute services
 
-The source cloud's control plane can be decomissioned,
-which is taking down only cloud controllers, database and messaging nodes.
-Nodes that must remain functional are those running the `Compute`, `Storage`,
-or `Networker` roles (in terms of composable roles covered by
-{OpenStackPreviousInstaller} Heat Templates).
+You must decomission cloud Controller nodes, database nodes, and messaging nodes on the source cloud's control plane. Nodes that are running the Compute, Storage, or Networker roles must remain functional (in terms of composable roles covered by {OpenStackPreviousInstaller} Heat Templates).
```

Comment:
What is meant by "(in terms of composable roles covered by {OpenStackPreviousInstaller} Heat Templates)"?

Reply:
those are roles named in terms of tripleo